demos/instr.cc

```diff
@@ -96,7 +96,6 @@ void UnwindStackAndSlowlyReturnTo(const void *address) {
       "mov x11, sp\n"
       "dc civac, x11\n"
       "dsb sy\n"
-      "isb\n"
```

Comment:
Can you leave a comment here explaining what happened to warn future-us not to put an `ISB` here?

Reply:
Done.